Formula/k/kosli-cli.rb

```diff
@@ -0,0 +1,31 @@
+class KosliCli < Formula
+  desc "CLI for managing Kosli"
+  homepage "https://docs.kosli.com/client_reference/"
+  url "https://github.com/kosli-dev/cli.git",
+      tag:      "v2.6.5",
+      revision: "8f4dc320cd81002b2e2f83516e679f3b0e685168"
+  license "MIT"
+  head "https://github.com/kosli-dev/cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/kosli-dev/cli/internal/version.version=#{version}
+      -X github.com/kosli-dev/cli/internal/version.gitCommit=#{Utils.git_head}
+      -X github.com/kosli-dev/cli/internal/version.gitTreeState=clean
+    ]
+    system "go", "build", *std_go_args(output: bin/"kosli", ldflags: ldflags), "./cmd/kosli"
+
+    generate_completions_from_executable(bin/"kosli", "completion", base_name: "kosli")
+  end
+
+  test do
+    output = shell_output("#{bin}/kosli version")
+    assert_match version.to_s, output
+    assert_match "go#{Formula["go"].version}", output
```

Comment:
This outputs the build-time `go` version, which is not always the same as the version of our `go` formula. It will likely fail when `go` is bumped to a newer version. If that's not desired, I think it's good enough not to check the `go` version here at all.